test/test_jsonchecker.py

```diff
@@ -247,3 +247,15 @@ def test_pass03(self):
             b'{"JSON Test Pattern pass3":{"The outermost value":"must be '
             b'an object or array.","In this test":"It is an object."}}',
         )
+
+    def test_exc_position(self):
```

Comment:
Can you add more tests? 

I would also appreciate a few that compare this against the standard library behavior on some input both find invalid. Its behavior also needs to be documented in the `loads()` section.

I would say also test the `e.doc` property but as elsewhere I'd prefer to not deal with that.